src/server/rpc/procedures/iex-trading/iex-trading.js

```diff
@@ -94,6 +94,35 @@ StockConsumer.dailyPercentChange = function(companySymbol) {
         });
 };
 
+/**
+ * Get historical closing price information about the specified company
+ * @param {String} companySymbol Company stock ticker symbol
+ * @param {String} range Time period (e.g. 1m, 3m, 1y, etc) to retrieve prices for
```

Comment:
It might be nice to change the first of the month examples to be days since that is the last format (maybe `15d` or something)

Reply:
This one might need some discussion on how we want to handle it. The API doesn't allow for custom durations, so 1m is the smallest it goes. We could give a more specific list of which ones it supports or make it just select a larger range (e.g. 1m for 15d) and select the relevant part of it, which isn't much work. Which would you prefer?

Reply:
In that case, don't worry about it. Looking at the IEXTrading docs, it has an option for `1d` but it seems to be a special case which gives minute-level points which don't have a 'close' field. We probably don't need to support that in this PR.